### DISCUSSION TITLE
npm version

### DISCUSSION THREAD

**Discussion by pnkapadia64** [mozilla/hubs#4712] [General Questions & Discussion] [answered]
I get the warning `The package-lock.json file was created with an old version of npm, so supplemental metadata must be fetched from the registry.` and then an error `TypeError: Cannot convert undefined or null to object` while trying `npm ci` to install hubs locally. What versions of npm are expected? And is there a registry that we may need to use?

Versions:
npm - 7.23.0
node - v14.17.3

[Edit]
After a successful `yarn install`, `pdfjs` is giving an error
```
ERROR in ./node_modules/pdfjs-dist/build/pdf.js 2780:70
Module parse failed: Unexpected token (2780:70)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|         }
| 
>         if (typeof desc.value === "function" && !value.hasOwnProperty?.(i)) {
|           continue;
|         }
 @ ./src/components/media-pdf.js 5:0-31 29:0-5 117:28-33
 @ ./src/hub.js
```

**Comment by pattersonbl2** [comment, marked as answer]
Was it a direct clone from the master branch of the repo ?  I haven't run into this issue before in my local installs.

**Reply by pnkapadia64** [reply]
Direct clone of the forked repo. Maybe downgrading my `npm` would help if you don't use v7.

**Reply by pattersonbl2** [reply]
I am currently running 7.21.1 without issues. you could try to downgrade to see if that could be the issue.

**Reply by pnkapadia64** [reply]
Ok, it worked with npm@7.21.1 and node@v12.16.2. Thank you
